pom.xml

```diff
@@ -35,6 +35,7 @@
   <!-- Modules are sorted in preferred build order. -->
   <modules>
     <module>optaweb-vehicle-routing-backend</module>
+    <module>optaweb-vehicle-routing-benchmark</module>
```

Comment:
May I suggest we don't call this a "benchmark"?

In the context of OptaPlanner, "benchmark" already has a very specific meaning - the result of the OptaPlanner benchmarker. In fact, Optaweb Employee Rostering does have a module called "benchmark", and it contains exactly this other kind of benchmark. In my opinion, this naming here causes needless confusion.

Also, I don't think that this module needs to exist at all. It's a very specific, one-off thing that might as well have its own repo. And that repo may even be a personal one, not a `kiegroup` one.

Reply:
### Naming

I can rename it to `benchmarks` so that it can accommodate benchmarks for other components in the future. My decision was to make that decision later when the need comes.

In my opinion it's a `benchmark` module no matter if it contains a benchmark for a single component or many components or even different types of benchmarks (micro, macro).

### Home
We're going to share the data next week with a specific group of people. They might want to run the benchmark themselves, so the code should be accessible somewhere.

I'm thinking about trying different GraphHopper modes to see if we can tweak it to get better performance. In that situation, it will come handy. Someone else might get into the same situation so I prefer to have it available in the upstream repo instead of a "random" fork.

We could use a feature branch but the code won't be maintained there so you'd have to rebase on master when you wanted to run the benchmark. The code will be always up to date and ready to run on `master` on the other hand.

### Productization
Everything that lands on master will be eventually shipped with the product. Is there a problem with shipping this? If yes, we can put it on a branch or tweak distribution assembly to exclude this module.

Reply:
But yes, I get your point. From a point of view this is an almost one-time thing that will maybe be used by me a couple of times and adds a little value to the project and some maintenance cost. We don't have to merge all of the code that has been written.

If there are strong concerns about adding this to master now, I'll close this and push the code to a feature branch so that it can be shared. We can decide whether it should go to master later.

Reply:
To clarify my opinion: I don't see a reason to add this to this repo at all. It's a Maven module - you can keep it as a Maven project in your own Github profile. That way, it can be shared with anyone who wants to use it, and doesn't have any impact on what we ship.

Reply:
Lukas, do you see any good reasons not to add distance matrix benchmark code to this repo? I can feel you're against that idea but I still haven't understood your motivation. What's the risk or cost of landing this on `master`.

Reply:
- It is one-off code, with no real product demand driving this. It means maintenance forever, with no one asking for it. It is a self-inflicted burden. I have several useful tools I use for my work in my own profile, and I understand they should not be merged, as they are too specific for a particular purpose. (Case in point: https://github.com/triceo/optaplanner-score-director-factory-benchmark)
- If it were to become an official benchmark, it'd have to be somewhat more scientific and reproducible. As we see in other comments here, this one does not provide consistent results on repeated runs.
- The naming (`benchmark`) is confusing, in the context of the wider OptaPlanner project.